### PR TITLE
[11.0] [MIG] crm_operating_unit

### PR DESCRIPTION
**Pull request by bjeficent**
Migration of crm_operating_unit to version 11.0

### CODE REVIEW

**Review by danypr92 — APPROVED** [review]
Great job! Thanks!

File: crm_operating_unit/models/crm_lead.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# © 2019 Eficent Business and IT Consulting Services S.L. -
```
Comment (nikul-serpentcs) [inline crm_operating_unit/models/crm_lead.py]: @bjeficent © / Copyright

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Comment by bjeficent** [comment]
Comments attended, thanks for reviewing.

**Review by nikul-serpentcs — APPROVED** [review]
Only Code review LGTM :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**